viewing_party/party.py

```diff
@@ -1,23 +1,174 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+
+    #Checks if any of the dictionary keys are None
+    if title == None or genre == None or rating == None:

```

Comment:
Note that the requirement is: 

If `title` is `falsy`, `genre` is `falsy`, or `rating` is `falsy`, this function should return `None`

Which means your code would allow a value like `""` for `title` and `genre` to be set in the dictionary. To implement the correct behavior, you will need to use the `not` keyword:

```suggestion
    if not title or not genre or not rating:
```

We should use `is` (and `is not`) for checks against singleton values like `None`, `True`, and `False` instead of `==`.

The reason is that is checks for object identity, ensuring we're comparing to the exact singleton instance (e.g., the one and only `None` object), not just something that happens to be equal in value.